.github/workflows/ci-pr.yaml

```diff
@@ -17,6 +17,10 @@ on:
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - '**/README.md'
+      - 'terraform/**'
+      - '.github/workflows/terraform-validate-ci.yaml'
```

Comment:
Question:
Is there a reason we don't add this to [/.github/workflows/ci-master.yaml](https://github.com/GoogleCloudPlatform/microservices-demo/blob/43b7ef46d90ff8bba7d0f2681792db4447232dd0/.github/workflows/ci-master.yaml)?

Reply:
Good point/catch! I thought I had a reason, but actually that totally makes sense to have this in there too! Working on that.

Reply:
Just took into account this. Note that I also renamed `ci-master.yaml` to` by `ci-main.yaml`, just heads up.